kuma/users/stripe_utils.py

```diff
@@ -67,3 +70,16 @@ def retrieve_stripe_subscription_info(user):
         }
 
     return None
+
+
+def create_missing_stripe_webhook():
+    url = absolutify(reverse("users.stripe_payment_succeeded_hook"))
```

Comment:
Something that would make this much more pleasant would be if I could put my `d847e2e5.ngrok.io` into my `.env` file and then I wouldn't even have to go to the Stripe.com dashboard to do anything. 
One thing that tripped me up was that this code here already created a webhook in Stripe on `http://localhost.org:8000...`

Reply:
True, good point. Thanks for making it! I've added a config key for a custom hostname and changed the docs accordingly.